### PR TITLE
docs: Complete missing parts and update outdated sections in Readme.md

### LINKED ISSUE #969: Complete missing parts and update outdated sections in Readme.md
## Description
As a Contributor,
I need an up-to-date README tells me everything about the project,
so that I can Contribute accordingly.

## To Do: 
1) Update the Features section, it seems to be written during project inception (2014).
2) Update the GSOC development section, specifically, Developments done in 2018 are missing.
3) Update the support section, it mentions a mailing list from systers.org.
4) Link to contributing guidelines doc in the Contributing Section.

## Mocks
The current feature section:
Screenshot from 2020-05-11 00-29-42

The Other 3 sections:
Screenshot from 2020-05-11 00-29-47

As seen, all the mentioned sections need to be updated.

### Update [Required]
- [ ] [Features section]
- [ ] [GSoC Development section]
- [ ] [Support section]
- [ ] [Contributing section]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-4 hours

### PR DESCRIPTION
**Pull request by gaurivn**
# Description
Completed missing parts and updated outdated sections in Readme.md.

Fixes #969 

# Type of Change:
**Delete irrelevant options.**

- Documentation

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ reporting.
...
 Features
 --------
 
-The VMS will be developed in phases. The initial implementation will track
+The VMS is developed in phases. The initial implementation will track
 volunteers' contact information, enable administrators to track volunteer
 hours, and allow reporting of useful information.
 
-The [requirements document](docs/Systers_GSoC14_VMS_Requirements.pdf) gives
+The [requirements document](https://github.com/anitab-org/vms/blob/develop/docs/GSoC18.md) gives
```
Comment (abha224) [inline README.md:25]: This doc does not have info about the scope of the project. Removing this sentence would be better.

Comment (gaurivn) [reply README.md]: Done, I removed

**Comment by gaurivn** [comment]
Thank you @abha224 for the review, I made the changes accordingly.

File: README.md [hunk 2]
```diff
@@ -86,6 +83,17 @@ Contribute
...
+## Branches
+
+The repository has the following permanent branches:
+
+ * **master** This contains the code which has been released.
+
+ * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch.
+
+ * **aws** 
+
```
Comment (abha224) [inline README.md:95]: This is the branch through which the project is deployed

Comment (gaurivn) [reply README.md:95]: I added it.

**Comment by abha224** [comment]
@anitab-org/vms-managers  I request one of you to provide a second review for this PR.

**Comment by yashshah2820** [comment]
@gaurivn can you squash the commits

File: README.md [hunk 3]
```diff
@@ -86,6 +83,17 @@ Contribute
   automatically closes the issue if your pull request gets merged.
 
 
+## Branches
+
+The repository has the following permanent branches:
+
+ * **master** This contains the code which has been released.
```
Comment (SanketDG) [inline README.md:90]: What is the meaning of "released" here?

Unless we actually follow a release cycle, this seems redundant.

Comment (mayburgos) [reply README.md:90]: It's the same as deployed. What's on this website: http://ec2-52-53-177-18.us-west-1.compute.amazonaws.com/en-us/

File: README.md [hunk 4]
```diff
@@ -101,19 +109,14 @@ Google Summer of Code (GSoC) Development
 We are pleased to participate in the Google Summer of Code and are grateful
 for the contributions from our GSoC developers.
 
+* [Completed features during GSoC 2018](https://github.com/anitab-org/vms/blob/develop/docs/GSoC18.md)
 * Completed features during GSoC 2016
        * [Development](https://github.com/systers/vms/wiki/GSoC'16---Migrating-VMS-from-Function-based-views-to-Class-based-views)
```
[the remark this answers is not in the document]
Comment (gaurivn) [reply README.md:114]: Can you elaborate?

[the remark this answers is not in the document]
Comment (SanketDG) [reply README.md:114]: Apologies. I meant:
```suggestion
       * [Development](https://github.com/systers/anitab-org/wiki/GSoC'16---Migrating-VMS-from-Function-based-views-to-Class-based-views)
```

**Pull request merged**